File: server-scala-play-server/app/model/TutorialResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for TutorialResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class TutorialResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  id: Option[Long],
  name: Option[String],
  description: Option[String],
  alignment: Option[String],
  image: Option[AssetShortResponse],
  orderIndex: Option[Long],
  objectType: Option[String],
  returning: Option[String]
)

object TutorialResponse {
  implicit lazy val tutorialResponseJsonFormat: Format[TutorialResponse] = Json.format[TutorialResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/TutorialResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class TutorialResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string Alignment { get; set; }
    public AssetShortResponse Image { get; set; }
    public long OrderIndex { get; set; }
    public string ObjectType { get; set; }
    public string Returning { get; set; }
}


